src/components/SearchList.js

```diff
@@ -1,33 +1,112 @@
 import React from 'react';
-import {View, Text, StyleSheet} from 'react-native';
-import {Card} from 'react-native-elements';
+import {View, Text, StyleSheet, ScrollView, ImageBackground} from 'react-native';
+import {Card, ListItem, Button, Tile, List, Image, Badge, Icon} from 'react-native-elements';
 // 3rd party libraries - additional
-
 // styles and language
-
-// our components - core
+import { badgeGreen, limeGreenTitle} from '../styles/index';
 // our components - additional
 
-const SearchList = ({sites}) => {
 
+const SearchList = ({sites}) => {
+  const{textStyle, IconContainer, subtitleView,siteAvatarStyle, siteAvatarContainerStyle, titleView, nearestTownStyle, badgeWrapperStyle, badgeContainerStyle} = styles
+  if(!sites){
+    return( <Text> No Sites Available </Text>)
+  }else{
     return (
-        <Card>
-            <Text>
-                List Results of Sites
-            </Text>
-        </Card>
-    );
+  <ScrollView>
+     <List>
+       {
+         sites.map(site => {
+            const {id, title, description, nearestTown} = site
+            return (
+                <ListItem
+                  key={id}
+                  title={
+                         <View>
+                           <View style={IconContainer}>
+                            <Icon
+                              name='image-inverted'
+                              type="entypo"
+                              color='purple' />
+                              <Icon
+                               name='air'
+                               type="entypo"
+                                color='#f50' />
+                              <Icon
+                                name='drop'
+                                type="entypo"
+                                color='#7C9EC2' />
+                                <Icon
+                                name='leaf'
+                                type="entypo"
+                                color='orange' />
+                             </View>
+                              <Badge
+                                wrapperStyle={badgeWrapperStyle}
+                                containerStyle={badgeContainerStyle}
+                                >
+                                  <Text style={nearestTownStyle}>{nearestTown}</Text>
+                              </Badge>
 
+                            <Text style={titleView}>{title}</Text>
+                          </View>
+                        }
+                  subtitle={
+                            <View style={subtitleView}>
+                                <Text>{description}</Text>
+                            </View>
+                           }
+                  avatar={require('../../assets/starTent.jpg')}
+                  avatarContainerStyle={siteAvatarContainerStyle}
+                  avatarStyle={siteAvatarStyle}
+                />
+               );
+             })
+           }
+       </List>
+     </ScrollView>
```

Comment:
For this list item, we should probably make it its own component so we can use it in other places in the application. Would you mind refactoring it to its own component, something like `SiteListItemCard` or something? Then you  can just import and use it here, and we can also use that `SiteListItemCard` other places.